pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -34,18 +34,17 @@ type PXCScheduledBackup struct {
 	Schedule           []PXCScheduledBackupSchedule  `json:"schedule,omitempty"`
 	Storages           map[string]*BackupStorageSpec `json:"storages,omitempty"`
 	ServiceAccountName string                        `json:"serviceAccountName,omitempty"`
-	Resources          *PodResources                 `json:"resources,omitempty"`
+	Resources          *corev1.ResourceRequirements  `json:"resources,omitempty"`
+	SchedulerName      string                        `json:"schedulerName,omitempty"`
+	PriorityClassName  string                        `json:"priorityClassName,omitempty"`
+	NodeSelector       map[string]string             `json:"nodeSelector,omitempty"`
```

Comment:
Resources, SchedulerName, PriorityClassName, NodeSelector also part of the storage